app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -1,94 +1,20 @@
 import { useI18n } from '@18f/identity-react-i18n';
-import { PageHeading, SpinnerDots } from '@18f/identity-components';
-import { request } from '@18f/identity-request';
+import { PageHeading } from '@18f/identity-components';
 import BackButton from './back-button';
-import LocationCollection from './location-collection';
-import LocationCollectionItem from './location-collection-item';
-import AnalyticsContext from '../context/analytics';
 import AddressSearch from './address-search';
-import InPersonContext from '../context/in-person';
+import InPersonLocations from './in-person-locations';
 
-interface PostOffice {
-  address: string;
-  city: string;
-  name: string;
-  phone: string;
-  saturday_hours: string;
-  state: string;
-  sunday_hours: string;
-  weekday_hours: string;
-  zip_code_4: string;
-  zip_code_5: string;
-}
-
-interface FormattedLocation {
-  formattedCityStateZip: string;
-  id: number;
-  name: string;
-  phone: string;
-  saturdayHours: string;
-  streetAddress: string;
-  sundayHours: string;
-  weekdayHours: string;
-}
-interface LocationQuery {
+interface InPersonLocationQuery {
   streetAddress: string;
   city: string;
   state: string;
   zipCode: string;
 }
 
-export const LOCATIONS_URL = '/verify/in_person/usps_locations';
-
-const getUspsLocations = (address) =>
-  request(LOCATIONS_URL, {
-    method: 'post',
-    json: { address },
-  });
-
-const formatLocation = (postOffices: PostOffice[]) => {
-  const formattedLocations = [] as FormattedLocation[];
-  postOffices.forEach((po: PostOffice, index) => {
-    const location = {
-      formattedCityStateZip: `${po.city}, ${po.state}, ${po.zip_code_5}-${po.zip_code_4}`,
-      id: index,
-      name: po.name,
-      phone: po.phone,
-      saturdayHours: po.saturday_hours,
-      streetAddress: po.address,
-      sundayHours: po.sunday_hours,
-      weekdayHours: po.weekday_hours,
-    } as FormattedLocation;
-    formattedLocations.push(location);
-  });
-  return formattedLocations;
-};
-
-const snakeCase = (value: string) =>
-  value
-    .split(/(?=[A-Z])/)
-    .join('_')
-    .toLowerCase();
-
-// snake case the keys of the location
-const prepToSend = (location: object) => {
-  const sendObject = {};
-  Object.keys(location).forEach((key) => {
-    sendObject[snakeCase(key)] = location[key];
-  });
-  return sendObject;
-};
-
 function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, registerField }) {
   const { t } = useI18n();
-  const [locationData, setLocationData] = useState([] as FormattedLocation[]);
-  const [foundAddress, setFoundAddress] = useState({} as LocationQuery);
-  const [inProgress, setInProgress] = useState(false);
-  const [autoSubmit, setAutoSubmit] = useState(false);
-  const [isLoadingComplete, setIsLoadingComplete] = useState(false);
-  const { setSubmitEventMetadata } = useContext(AnalyticsContext);
-  const { arcgisSearchEnabled } = useContext(InPersonContext);
```

Comment:
It's not safe to remove a feature flag in the same story as you're making significant refactors or functionality changes. I also don't see the removal of this flag specified in any of the story's AC.